src/hdmf/common/resources.py

```diff
@@ -131,40 +158,56 @@ class ObjectKey(Row):
     __table__ = ObjectKeyTable
 
 
-@register_class('ExternalResources', EXP_NAMESPACE)
+@register_class("ExternalResources", EXP_NAMESPACE)
 class ExternalResources(Container):
     """A table for mapping user terms (i.e. keys) to resource entities."""
 
     __fields__ = (
-        {'name': 'keys', 'child': True},
-        {'name': 'files', 'child': True},
-        {'name': 'objects', 'child': True},
-        {'name': 'object_keys', 'child': True},
-        {'name': 'entities', 'child': True},
+        {"name": "keys", "child": True},
+        {"name": "files", "child": True},
+        {"name": "objects", "child": True},
+        {"name": "object_keys", "child": True},
+        {"name": "entities", "child": True},
     )
 
-    @docval({'name': 'keys', 'type': KeyTable, 'default': None,
-             'doc': 'The table storing user keys for referencing resources.'},
-            {'name': 'files', 'type': FileTable, 'default': None,
-             'doc': 'The table for storing file ids used in external resources.'},
-            {'name': 'entities', 'type': EntityTable, 'default': None,
-             'doc': 'The table storing entity information.'},
-            {'name': 'objects', 'type': ObjectTable, 'default': None,
-             'doc': 'The table storing object information.'},
-            {'name': 'object_keys', 'type': ObjectKeyTable, 'default': None,
-             'doc': 'The table storing object-resource relationships.'},
-            {'name': 'type_map', 'type': TypeMap, 'default': None,
-             'doc': 'The type map. If None is provided, the HDMF-common type map will be used.'},
-            allow_positional=AllowPositional.WARNING)
+    @docval(
+        {
+            "name": "keys",
+            "type": KeyTable,
+            "default": None,
+            "doc": "The table storing user keys for referencing resources.",
+        },
+        {
+            "name": "files",
+            "type": FileTable,
+            "default": None,
+            "doc": "The table for storing file ids used in external resources.",
+        },
+        {"name": "entities", "type": EntityTable, "default": None, "doc": "The table storing entity information."},
```

Comment:
These should be changed to match the same style as the others

Reply:
This comment applies to the rest of the file to avoid repeating the same comment